deploy/roles/postgresql-server/templates/pg_ident.conf.j2

```diff
@@ -0,0 +1,3 @@
+{% for ident in pp_ident_mappings %}
```

Comment:
Can we put some explanatory comments at the top of the file? e.g.:
`# This is a list of identity mappings for use with Kerberos; cf. some-url-or-other`
`# columns: Identity Type, Identity Match, Postgres Role`

Reply:
Done, thanks!